lib/pbench/server/api/resources/upload_api.py

```diff
@@ -40,6 +38,9 @@ def get(self):
 
 
 class Upload(Resource):
+    ALLOWED_EXTENSION = ".tar.xz"
+    CHUNK_SIZE = 65536
```

Comment:
Since you've tweaked the spelling of `CHUNK_SIZE` on agent and server side ... makes me wonder whether we really need to bump this into a common constants module so they're guaranteed to be in sync. (Well, "mostly guaranteed" to be in sync, since agent and server are still built separately...)

Reply:
I am not in favor of explicitly tying the agent and server to the same size.  We might want even larger buffers on the server side in the future to take advantage of available memory.